frontend/package-lock.json

```diff
@@ -30,21 +30,35 @@
         "zustand": "^4.3.2"
       },
```

Comment:
are you sure that all of these changes are not related to local development changes? Cause, if that happens, the production deploy could result broken